script/common.sh

```diff
@@ -45,3 +45,13 @@ function check_changed_packages() {
   fi
   return 0
 }
+
+# Activates the Flutter plugin tool.
+# This ensures that the plugin tools dependencies are resolved using the current Dart SDK.
+function activate_plugin_tool() {
+  if [ "$(expr substr $(uname -s) 1 5)" == "MINGW" ]; then
```

Comment:
We should extract this from the locations it's repeated to a common.sh utility that returns the pub name.

Reply:
Done